application/server/server.go

```diff
@@ -197,7 +197,7 @@ func initializeHandler(srv *jrpc2.Server, c *config.Config) handler.Func {
 
 		c.SetClientCapabilities(params.Capabilities)
 		setClientInformation(params)
-		di.Analytics().Initialise()
+		di.Analytics().Initialise() //nolint:misspell // breaking api change
```

Comment:
Should we just disable the spelling linter? I already hate it ;).

Reply:
The payoff will be future consistency.

Reply:
I really don't think linting should include spell-checks.